### PR TITLE
Add commands and storage

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

File: src/main/java/world/bentobox/border/Border.java [hunk 1]
```diff
@@ -1,36 +1,120 @@
...
+        if (data == null) {
+            return;
+        }
+        new BukkitRunnable() {
+
+            @Override
+            public void run() {
+                handler.saveObject(data);
+            }
+        }.runTaskAsynchronously(getPlugin());
```
Comment (tastybento) [inline src/main/java/world/bentobox/border/Border.java]: SaveObject is async so this wouldn't be required. Doesn't harm though.

File: src/main/java/world/bentobox/border/Border.java [hunk 2]
```diff
@@ -1,36 +1,120 @@
...
+    }
+
+    public void updateBorder(Player player, Location location) {
+        removeBorder(player); // Remove current
+        getLevelsData(player.getUniqueId(), data -> {
+            if (!data.isEnabled()) {
+                return;
+            }
+            Island island = getPlugin().getIslands().getIslandAt(location).orElse(null);
```
Comment (tastybento) [inline src/main/java/world/bentobox/border/Border.java]: You can also just do:

```
getPlugin().getIslands().getIslandAt(location).ifPresent(island -> worldBorderApi.setBorder(player, island.getProtectionRange() * 2, island.getCenter()));
```

**Comment by sgdc3** [comment]
I'm currently running it on my server without issues

File: src/main/java/world/bentobox/border/Border.java (hunk 1 above)
Comment (sgdc3) [reply src/main/java/world/bentobox/border/Border.java]: Nah, I'll remove it as it's a waste of resources

File: src/main/java/world/bentobox/border/Border.java (hunk 2 above)
Comment (sgdc3) [reply src/main/java/world/bentobox/border/Border.java]: I know, I personally try to avoid optionals in simple scenarios

**Comment by tastybento** [comment]
Thanks!

**Pull request merged**